devtools-ci-index.yaml

```diff
@@ -1633,6 +1633,23 @@
 
     <<: *eclipse-che-automation-template
 
+- job-template:
+    name: '{ci_project}-{git_repo}-che-pullrequests-test'
+    wrappers:
+    - vault-secrets:
+        <<: *vault_defaults
+        secrets: 
+            - *quay-eclipse-che-credentials
+    scm:
+        - git:
+            url: https://github.com/{git_organization}/{git_repo}.git
+            shallow_clone: true
+            branches:
+                - cico-pr-test
+    triggers:
+        - github
```

Comment:
Trigger should be [github pull request](https://docs.openstack.org/infra/jenkins-job-builder/triggers.html#triggers.github-pull-request) instead of [github](https://docs.openstack.org/infra/jenkins-job-builder/triggers.html#triggers.github).

When you will change that to use github pull request builder plugin for triggering this job, please read documentation of this plugin and make changes according to this (https://wiki.jenkins.io/display/JENKINS/GitHub+pull+request+builder+plugin).
Or you can inherit from some template, which is using [correct configuration](https://github.com/openshiftio/openshiftio-cico-jobs/blob/master/devtools-ci-index.yaml#L123)

Hope this helps ;-)

Afaik, this job as it is right now, would be triggered on push to branch named `cico-pr-test`.